components/ProgressIndicator/ProgressIndicator.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { number } from 'prop-types';
+import styles from './ProgressIndicator.css';
+
+const ProgressIndicator = ({ stepNumber, totalSteps }) => {
+  if (totalSteps < 1) return null;
+
+  const percentageCompleted = (stepNumber / totalSteps) * 100;
+
+  return (
+    <div className={styles.ProgressIndicator}>
```

Comment:
There's a semantic way to express progress indicators!

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress

Reply:
TIL, thanks for sharing Kyle.